tests/interop/test_add_function.cpp

```diff
@@ -26,6 +38,14 @@ int main(int argc, char **argv) {
     assert(wasmTable.length === beforeLength);
   });
 #endif
+  // We have a guarantee of index uniqueness for each
+  // function.
+  EM_ASM({
+    assert(wasmTable.length >= 3);
+    assert(addFunction(wasmTable.get(1)) === 1);
+    assert(addFunction(wasmTable.get(2)) === 2);
+  }, &foo, &bar, &baz);
```

Comment:
Are foo, baz, bar even used here ?

Reply:
They are not used inside the JS, but sending their addresses into JS ensures that their addresses are taken (in a way LLVM can't optimize out) and therefore that the Table is at least of that size.

Reply:
Can we do that more explicitly?

Reply:
Or at least add a comment.